code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -14,15 +14,15 @@
 	skinned_type = /obj/item/stack/sheet/metal // Let's grind up IPCs for station resources!
 
 	eyes = "blank_eyes"
-	brute_mod = 2.28 // 100% * 2.28 * 0.66 (robolimbs) ~= 150%
-	burn_mod = 2.28  // So they take 50% extra damage from brute/burn overall
+	brute_mod = 1.97 // 100% * 1.82 * 0.66 (robolimbs) ~= 130%
```

Comment:
Comment needs updating, also it's a good idea to actually specify the number changes in the PR description

Reply:
done